src/v/cluster/topics_frontend.cc

```diff
@@ -233,10 +233,43 @@ ss::future<std::error_code> topics_frontend::do_update_data_policy(
     }
 }
 
+ss::future<std::error_code> topics_frontend::do_update_replication_factor(
+  topic_properties_update& update, model::timeout_clock::time_point timeout) {
```

Comment:
can you take `update` by value? it looks like the caller only needs to make a copy of the topic-namespace when moving `update` into this method for that to be possible.